config_defaults.yml

```
@@ -556,46 +556,43 @@ Rakefile:
 Gemfile:
   required:
     ':development':
-      # hardcode JSON version to what's shipped in pdk for now
-      - gem: json
-        version: '= 2.0.4'
-        condition: "Gem::Requirement.create('~> 2.4.2').satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: json
-        version: '= 2.1.0'
-        condition: "Gem::Requirement.create(['>= 2.5.0', '< 2.7.0']).satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: json
-        version: '= 2.3.0'
-        condition: "Gem::Requirement.create(['>= 2.7.0', '< 2.8.0']).satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: 'puppet-module-posix-default-r#{minor_version}'
-        version: '~> 1.0'
+      - gem: 'json'
+      - gem: 'voxpupuli-puppet-lint-plugins'
+        version: '>= 3.0.0'
+      - gem: 'facterdb'
+        version: '< 2.0.0'
```

It'd strongly suggest a lower bound for this and other gems.
```suggestion
        version: '~> 2.0'
```

---

lower bounds added to everything